Orange/classification/simple_tree.py

```diff
@@ -93,9 +93,9 @@ def fit_storage(self, data):
 
 class SimpleTreeModel(Model):
     def __init__(self, learner, data):
-        X = np.ascontiguousarray(data.X)
-        Y = np.ascontiguousarray(data.Y)
-        W = np.ascontiguousarray(data.W)
+        X = np.ascontiguousarray(data.X.astype(np.float64))
+        Y = np.ascontiguousarray(data.Y.astype(np.float64))
+        W = np.ascontiguousarray(data.weights.astype(np.float64))
```

Comment:
Why `np.ascontiguousarray()`? It seems to be the default in `_to_numpy()`, making this a needless copy? Also the type should be `float` (=`np.float64`) anyway?